pkgs/by-name/repath-studio/package.nix

```diff
@@ -0,0 +1,201 @@
+        -name \*.lastUpdated \
+        -o -name resolver-status.properties \
+        -o -name _remote.repositories \) \
+        -delete
+
+      runHook postInstall
+    '';
+
+    dontFixup = true;
+
+    outputHash = "sha256-wytFeZkVE6zAivyTwF5wv8SgskocN33FzhojOjETWog=";
+    outputHashMode = "recursive";
+    outputHashAlgo = "sha256";
+  };
+
+  makeCacheWritable = true;
+
+  npmDeps = fetchNpmDeps {
+    name = "${finalAttrs.pname}-${finalAttrs.version}-deps";
+    inherit (finalAttrs) src;
+    hash = "sha256-/Wj//cxM3bhBeU8LIyA/fMO9MZHoNs15apKBzhx6sCQ=";
+  };
+
+  nativeBuildInputs =
+    let
+      clojureWithCache = writeShellScriptBin "clojure" ''
+        exec ${lib.getExe' clojure "clojure"} -Sdeps '{:mvn/local-repo "${finalAttrs.mavenRepo}"}' "$@"
+      '';
+    in
+    [
+      clojureWithCache
+      nodejs
+      npmHooks.npmConfigHook
+      makeWrapper
+      copyDesktopItems
+    ];
+
+  env = {
+    ELECTRON_SKIP_BINARY_DOWNLOAD = true;
+    PUPPETEER_SKIP_DOWNLOAD = true;
+  };
+
+  postPatch = ''
+    substituteInPlace shadow-cljs.edn \
+      --replace-fail ":shadow-git-inject/version" '"v${finalAttrs.version}"'
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    # electronDist needs to be modifiable on Darwin
+    cp -r ${electron.dist} electron-dist
+    chmod -R u+w electron-dist
+  ''
+  # Electron builder complains about symlink in electron-dist
+  + lib.optionalString stdenv.hostPlatform.isLinux ''
+    rm electron-dist/libvulkan.so.1
+    cp ${lib.getLib vulkan-loader}/lib/libvulkan.so.1 electron-dist
+  ''
+  + ''
+    npm run build
+    npm exec electron-builder -- --dir \
+      -c.electronDist=electron-dist \
+      -c.electronVersion=${electron.version}
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    ${
+      if stdenv.hostPlatform.isDarwin then
+        # bash
+        ''
+          mkdir -p $out/Applications
+          cp -r "dist/mac"*"/Repath Studio.app" "$out/Applications"
+          makeWrapper "$out/Applications/Repath Studio.app/Contents/MacOS/Repath Studio" "$out/bin/repath-studio"
+        ''
+      else
+        # bash
+        ''
+          mkdir -p $out/share/{repath-studio,icons/hicolor/scalable/apps}
+          cp -r dist/*-unpacked/resources/app.asar $out/share/repath-studio
+          cp resources/public/img/icon.svg $out/share/icons/hicolor/scalable/apps/repath-studio.svg
+
+          makeWrapper '${lib.getExe electron}' "$out/bin/repath-studio" \
+            --add-flags "$out/share/repath-studio/app.asar" \
+            --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
+            --set-default ELECTRON_FORCE_IS_PACKAGED 1 \
+            --inherit-argv0
+        ''
+    }
+
+    runHook postInstall
+  '';
+
+  # chromium package not available for darwin
+  doCheck = stdenv.hostPlatform.isLinux;
+  checkPhase =
+    lib.optionalString stdenv.hostPlatform.isLinux
```

Comment:
You don't need to explicitly specify this since it's already controlled by `doCheck`.

```suggestion
```